src/sentry/static/sentry/app/views/sharedGroupDetails/sharedGroupHeader.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import styled from '@emotion/styled';
+
+import EventMessage from 'app/components/events/eventMessage';
+import space from 'app/styles/space';
+import {Group} from 'app/types';
+import overflowEllipsis from 'app/styles/overflowEllipsis';
+
+type Props = {
+  group: Group;
+};
+
+const SharedGroupHeader = ({group}: Props) => (
+  <Wrapper>
+    <Details>
+      <Title>{group.title}</Title>
+      <EventMessage message={group.culprit} />
+    </Details>
+  </Wrapper>
+);
+
+export default SharedGroupHeader;
+
+const Wrapper = styled('div')`
+  padding: ${space(3)} ${space(4)} ${space(3)} ${space(4)};
```

Comment:
With all this CSS moving to styled components are we able to remove some CSS from the LESS files?

Reply:
I wanted too, but decided to do it when refactoring the component header.jsx